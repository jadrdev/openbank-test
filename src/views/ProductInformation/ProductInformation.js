import React from 'react'

import ImageInfoSnippet from "../../components/InfoSnippet/ImageInfoSnippet";

import "./ProductInformation.scss";
import mindImage from "../../assets/img/group.svg";
import safeImage from "../../assets/img/group-3.svg";
import InfoSnippet from "../../components/InfoSnippet/InfoSnippet";

export default function ProductInformation() {
    return (
        <div className="ProductInformation">
            <div className="first-two-info-snippets">
                <ImageInfoSnippet
                    style={{ width: "40%" }}
                    imageSource={mindImage}
                    imageAlt={"mind-openbank"}
                    infoSnippet="Guarda aquí todas tus contraseñas, datos o cualquier información, olvida las notas de papel y las aplicaciones no protegidas"
                />
                <ImageInfoSnippet
                    style={{ width: "40%" }}
                    imageSource={safeImage}
                    imageAlt="safe-openbank"
                    infoSnippet="Crea tu clave maestra: solo tú podrás acceder a tus secretos con ella."
                />
            </div>
            <InfoSnippet
                label="Cómo funciona"
                description="En primer lugar, debes crear una contraseña diferente para sus pertenencias electrónicas. No podrás recuperar tu contraseña, así que recuerdala bien"
            />
            <InfoSnippet
                label="Qué datos puedes guardar"
                description="Por ejemplo, el número de tu tarjeta, el PIN y el PUK de tu teléfono móvil, el número de serie de alguno de tus dispositivos o cualquier información que necesites tener en lugar seguro."
            />
        </div>
    )
}
