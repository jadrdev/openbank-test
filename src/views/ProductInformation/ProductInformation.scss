@import "../../styles/base";

.ProductInformation {
  display: flex;
  flex-direction: column;
  row-gap: 2em;
  height: 100%;
  .first-two-info-snippets {
    width: 100%;
    display: flex;
    align-items: center;
    justify-content: center;
  }
}