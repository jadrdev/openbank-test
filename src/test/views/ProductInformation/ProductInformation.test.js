import React from 'react'
import { shallow } from "enzyme";

import ProductInformation from "../../../views/ProductInformation/ProductInformation";

describe('Pruebas en el componente ProductInformation', () => {
    test('Debe de verse', () => {
        const wrapper = shallow(<ProductInformation />)
        expect(wrapper).toMatchSnapshot();
    })
});

