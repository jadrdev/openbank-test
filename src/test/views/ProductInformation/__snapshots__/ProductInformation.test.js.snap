// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente ProductInformation Debe de verse 1`] = `ShallowWrapper {}`;
